CHANGELOG.unreleased.md

```diff
@@ -34,6 +35,7 @@
 >
 > In case of conflict, the highest (lowest in previous list) `$version` wins.
 
+- xo-server-web-hooks minor
```

Comment:
This change is unrelated and should have been avoided :slightly_smiling_face:

Reply:
Do you mean it should be fixed in another PR?

Reply:
Either that or don't fixed as it's not a big issue :slightly_smiling_face: